### ISSUE TITLE
minSDKVersion

### ISSUE THREAD

**Issue by TurKurT656** [AleSpero/ExpandableCardView#16]
Can u change the minSDKVersion to at least 17? there is a pull request for that but it also changed that to 18

**Comment by AleSpero** [comment]
Hello TurKur,

I have merged the PR with minSdkVersion 18 and will release the new version soon.
In the next days i will properly test the library and check if it's possible to lower the minSdkVersion to 17.

Regards,

Alessandro

**Comment by Ayusch** [comment]
This issue is resolved. minSdkVersion is now 18. I guess it's low enough to support most of the features.

**Comment by TurKurT656** [comment]
I've tried to import but again it gives me the error again:
`Manifest merger failed : uses-sdk:minSdkVersion 18 cannot be smaller than version 21 declared in library [com.alespero:expandable-cardview:0.8] C:\Users\TurKurT656\.gradle\caches\transforms-1\files-1.1\expandable-cardview-0.8.aar\7c4cd474cd331366c34987112b387d13\AndroidManifest.xml as the library might be using APIs not available in 18`

**Comment by Ayusch** [comment]
See the source code : it's 18, but I guess the gradle has not been updated yet. So, you can wait for that, or just clone the repo, generate a .aar file and start using.

Hope that helps!!

**Comment by TurKurT656** [comment]
Yes it is. I guess if the version upgraded to 0.8.1 the problem will be solved
